docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```diff
@@ -451,6 +453,9 @@ code: |
   
   # if statement for idempotency - ensure tests are only run once
   if not has_run_tests:
+    # Get interview vars that are external to the background action
+    # up here to avoid repeating the block lower down.
+    env_vars
```

Comment:
Wasn't sure if this was necessary, but I did have some trouble previously with the code looping through the block multiple times despite using variables that I thought had already been satisfactorily defined.